code/modules/client/preference/link_processing.dm

```diff
@@ -941,7 +941,22 @@
 					if(ishuman(usr)) //mid-round preference changes, for aesthetics
 						var/mob/living/carbon/human/H = usr
 						H.remake_hud()
-
+				if("map_pick")
+					var/list/potential_maps = list()
+					map_vote_pref_json = list() //Clear it out
```

Comment:
This should only clear it out if they successfully change their map pref, no?

Reply:
done